apps/web/src/styles/7-components/sort-table.scss

```diff
@@ -0,0 +1,60 @@
+@use "../govuk-frontend/govuk/settings/_colours-applied" as colors;
+.sort-table {
+  &__button {
+	background-color: colors.govuk-colour('white');
```

Comment:
please use the available vars for colours

Reply:
?

This is following what file-list, app-header, infobox etc do.

Reply:
I referred to the whole file

box-shadow: 0 0 0 0;
color: #005ea5;

background-color: #fd0;
color: #0b0c0c;

Reply:
misunderstood. Will fix now.

Reply:
addressed